### PR TITLE
<AddressInput /> – New component added

### PR DESCRIPTION
**Pull request by MSiliunas**
### 🔦 Summary

Adds a new internal component - `<AddressInput />`.

### ✅ Checklist

- [x] 👨‍💻 API change is approved by the UI Infra Developers 
- [x] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [x] Story
  - [x] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [x] Component
  - [ ] Visual test

### CODE REVIEW

File: src/AddressInput/test/AddressInput.spec.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    const text = 'test';
+    const props = {
+      onChange: Sinon.spy(),
+    };
+    const { driver } = render(<AddressInput {...props} />);
+
+    await driver.enterText(text);
+
+    expect(props.onChange.called).toEqual(true);
+    expect(await driver.getInputValue()).toEqual(text);
```
Comment (EgoziE) [inline src/AddressInput/test/AddressInput.spec.js:39]: Might look a bit too much, but we encourage testing the before and after.

```suggestion
    expect(props.onChange.called).toEqual(false);
    expect(await driver.getInputValue()).toEqual('');

    await driver.enterText(text);

    expect(props.onChange.called).toEqual(true);
    expect(await driver.getInputValue()).toEqual(text);
```

File: src/AddressInput/test/AddressInput.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+function addressInputWithMandatoryProps() {
+  return <AddressInput />;
+}
+
+function addressInputWithAllProps() {
+  return (
+    <AddressInput
+      dataHook="dataHook"
+      className="className"
```
Comment (EgoziE) [inline src/AddressInput/test/AddressInput.tsx:17]: Type test
Just add all props available here

**Review by EgoziE — CHANGES_REQUESTED** [review]

File: perfer.config.js [hunk 3]
```diff
@@ -152,6 +152,7 @@ const javascriptFiles = [
   ['VerticalTabs.bundle.min.js', 38],
   ['VerticalTabsItem.bundle.min.js', 38],
   ['WarningIndicator.bundle.min.js', 40],
+  ['AddressInput.bundle.min.js', 10],
```
Comment (EgoziE) [inline perfer.config.js:155]: Change to 62 (according to the bundle size test) and keep sorted

File: perfer.config.js [hunk 4]
```diff
@@ -268,6 +269,7 @@ const cssFiles = [
   ['Tooltip.min.css', 2],
   ['VariableInput.min.css', 2],
   ['WarningIndicator.min.css', 2],
+  ['AddressInput.min.css', 1],
```
Comment (EgoziE) [inline perfer.config.js:272]: keep sorted

**Review by EgoziE — CHANGES_REQUESTED** [review]

File: src/AddressInput/AddressInput.js [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+    this.setState({
+      inputValue: value,
+    });
+    this._debouncedOnChange(value);
+  };
+
+  _onSelect = value => {
+    const { onSelect } = this.props;
+    this.setState({
+      inputValue: value.value,
```
Comment (lironch) [inline src/AddressInput/AddressInput.js:37]: Shouldn't this be `value.description`? assuming `description` will be part of the option object propTypes

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6049

**Review by EgoziE — APPROVED** [review]

**Pull request merged**